packaging/nuget/NServiceBus.Persistence.Sql.MsBuild.targets

```diff
@@ -1,12 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <ScriptDirectory>$(ProjectDir)$(IntermediateOutputPath)NServiceBus.Persistence.Sql\</ScriptDirectory>
```

Comment:
@MarcinHoppe `$(ProjectDir)$(IntermediateOutputPath)` dont exist until later in the build. i needed to move them down to get it working in the new project system